plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php

```diff
@@ -188,6 +189,54 @@ private function enable_order_email_actions_for_pos_orders() {
 			add_action( 'woocommerce_rest_order_actions_email_send', array( $this, 'trigger' ), 10, 2 );
 		}
 
+		/**
+		 * Override settings form fields to remove the enbaled/disabled field as the email is manually sent.
```

Comment:
```suggestion
		 * Override settings form fields to remove the enabled/disabled field as the email is manually sent.
```

Reply:
More a question for the future Flux reviewer, really... is there a better way to do this than fully specifying all the fields?

For example, can we add an action which updates `form_fields` to remove the enabled/disabled flag?

Reply:
Fixed typo in https://github.com/woocommerce/woocommerce/pull/58012/commits/3d18c01c5d43d1c1fead86ff48d78e5b0bddab9c.

> More a question for the future Flux reviewer, really... is there a better way to do this than fully specifying all the fields?

I followed the same approach as in the only manually triggered core email "customer invoice":

https://github.com/woocommerce/woocommerce/blob/f29d45c1f7e4fdd8c4191ca03f165ce3a028612b/plugins/woocommerce/includes/emails/class-wc-email-customer-invoice.php#L198-L257

Will tag the Flux reviewer about this question.

Reply:
Tagging @adimoldovan for the question about a better way to hide certain default email form fields.

Reply:
> Tagging @adimoldovan for the question about a better way to hide certain default email form fields.

I'm removing myself and adding @albarin as reviewer from Flux for a more experienced set of eyes on this PR.